app/controllers/backdate_controller.rb

```diff
@@ -23,4 +23,15 @@ def update
       redirect_to document_path(edition.document)
     end
   end
+
+  def destroy
+    result = Backdate::DestroyInteractor.call(params: params, user: current_user)
+
+    if result.api_error
```

Comment:
As this interactor can't return an api_error we don't need this code.

Reply:
Removed